src/transform/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+catkin_install_python(PROGRAMS dive_effort_to_twiststamped.py
```

Comment:
Do these need to be installed? What is the reason for doing that instead of just running the code out of the catkin_ws